service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseManagementOrder.java

```diff
@@ -33,7 +34,7 @@
 
     @JsonIgnore
     public boolean isDraft() {
-        return action == null || !action.getType().equals(SEND_TO_ALL_PARTIES);
+        return ofNullable(action).map(x -> !SEND_TO_ALL_PARTIES.equals(action.getType())).orElse(true);
```

Comment:
Not sure then conversion to the optional improves anything although I would agree that the flipping of the equals is better

Reply:
If you want to be insistent on the optional then it would be better to use x in the map function rather than action

Reply:
fine to revert and then swap the equals around